src/SFA.DAS.ApplyService.Domain/Ukrlp/ProviderDetails.cs

```diff
@@ -68,6 +68,34 @@ public class ContactAddress
         public string Address4 { get; set; }
         public string Town { get; set; }
         public string PostCode { get; set; }
+
```

Comment:
not for here, but we may need to move this kind of thing closer to the view.  Let's discuss with Greg some time...

Reply:
Yeah I'll move the format function into the VM